clocks/clock-09.js

```diff
@@ -52,15 +45,51 @@ var clock09 = function(sketch) {
     for (let i = 0; i < hour.length; i++){
        drawCharacter(hour[hour.length -1 - i], sketch.width/2 - (spacing * (i + 1) * 6), sketch.height/2,size);
     }
-
     // Draw the : in between hour and minute
     drawCharacter(":", sketch.width/2, sketch.height/2,size);
 
     // For each character in minute, draw that character
     for (let i = 0; i < minute.length; i++){
        drawCharacter(minute[i], sketch.width/2 + (spacing * (i+1) * 6) , sketch.height/2,size);
-
     }
   }
-
-}
+  function drawClock(x,y,w,h)
+  {
+    let hr = sketch.hour();
+    let mn = sketch.minute();
+    let sc = sketch.second();
+  	let secondAngle = sketch.map(sc, 0, 60, 0, 360) - 90;
+  	let minuteAngle = sketch.map(mn, 0, 60, 0, 360) -90;
+    let hourAngle = sketch.map(hr % 12, 0, 12, 0, 360) -90;
+    sketch.noStroke();
+    // Draw the outer circle
+    sketch.fill(224);
+    sketch.ellipse(x,y,w,h);
+    // Draw the inner cirle
+    sketch.fill(255);    
+    sketch.ellipse(x,y,w-1,h-1);
+    // Draw the arms / arcs
+    sketch.push();
+      sketch.noStroke();
+      sketch.fill(0);
+      sketch.translate(x,y);
+      // Draw the hour arm
+      sketch.push();
+        sketch.rotate(hourAngle);
+        sketch.triangle(0,0,1.8,-0.5,1.8,0.5);
```

Comment:
Is it possible to get these to scale correctly with the clocks? The clocks may be able to be viewed larger so using these constant causes some issues - and I am not sure the best way to fix? I'll merge this anyway but it would be good to see that!